visualize_threads.py

```diff
@@ -34,13 +56,19 @@ def onClick(event):
 
 # animation update function
 def updatefig(*args):
-    global vis_data, data, i, vis_rows, im, rows, ani, pause
+    global vis_data, data, i, vis_rows, im, rows, ani, pause, annotate
     try:
         vis_data[:,:] = data[i:i + vis_rows, :]
     except:
         plt.close(fig)
         sys.exit()
 
+    if(annotate and data[i,-1] == 1): 
+        plt.title("Positive Annotation")
+    else: 
+        plt.title("Negative Annotation")
+
```

Comment:
3 things:
1. simplify this whole if else to `plt.title("Positive Annotation" if annotate and data[i, -1] == 1 else "Negative Annotation")`
2. also won't this show "negative annotation" when we don't have any annotations at all (ie: displaying a file from data/converted/...)
3. this will update the plot ui title with every single row being processed - seems a bit excessive because this would run `plt.title("Positive Annotation")` when the plot title is already "Positive Annotation". Let's keep a global boolean variable or something to prevent updating the title unnecessarily